.github/workflows/action-test.yml

```
@@ -22,14 +22,14 @@ on:
   push:
     branches:
     - master
-    - github-actions2
   # run the test only if the PR is to master
   # turn it on if required
   #pull_request:
   #  branches:
   #  - master
   schedule:
-    - cron: '0 0 * * *'
+    # - cron: '0 0 * * *'  # nightly
+    - cron: '0 */3 * * *'  # every 3 hours
```

Are you going to change this back? Otherwise I'd suggest to just replace the line:

```suggestion
    - cron: '0 */3 * * *'  # every 3 hours
```

---

yeah no, gonna change it back, we really don't want to run that test every 3h for the rest of our time :grin: